petab/petab_schema.yaml

```diff
@@ -0,0 +1,46 @@
+# For syntax see: https://json-schema.org/understanding-json-schema/index.html
+#$schema: "https://json-schema.org/draft/2019-09/meta/core"
+$schema: "http://json-schema.org/draft-06/schema"
+description: PEtab parameter estimation problem config file schema
+
+properties:
+
+  petab_version:
+    type: string
+    description: Version of the PEtab format (e.g. 0.0.0a16)
+
+  parameter_file:
```

Comment:
we discussed also allowing for multiple parameter files, and then doing a (n interactive) merge. one could have that as part of the standard, or have a helper function that creates a petab conform single parameter_file. opinions?

Reply:
in the prior case, there might be an alternative parameter_file field under problems.

Reply:
I think we don't have that use case yet. I would keep it for a later version.

Merging parameter files: Strongly in favor of putting it in a separate function. Will leave implementation to first person having a use (and test) case for that. Will create an issue.